.github/workflows/helm-release.yml

```
@@ -0,0 +1,33 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "deployment/helm/**/Chart.yaml"
```

I was looking into some examples, maybe add as well
```suggestion
      - "deployment/helm/**/Chart.yaml"
      - "deployment/helm/**/values.yaml"
      - "deployment/helm/**/templates/**"
```

---

Mixed feelings here, we have a Helm test which will always validate the bump of the version in `Chart.yaml`. If by some reason, that fails and any reviewer forgets to double-check it, a merge can try to create a release that already exists (since `version` in `Chart.yaml` was not updated).

I think it is safer to keep only the helm release running when `Chart.yaml` has changes. WDYT?

---

ok makes sense, will resolve